SECP-service/src/main/resources/assets/app/js/app.js

```diff
@@ -1,28 +1,28 @@
 // Declare app level module which depends on filters, and services
 angular.module('SECP', ['ngResource', 'ngRoute', 'ui.bootstrap', 'ui.date',
```

Comment:
I see that you addressed the spacing issue in this file. Please kindly verify that nothing has changed.